src/js/components/AnalysisPanel/AnalysisFormElements/AnalysisRangeSlider.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+
+import Slider from 'rc-slider';
+import MapActions from '../../../actions/MapActions';
```

Comment:
Any reason we're using the ../../../ pathing rather than the 

import mapActions from 'actions/MapActions';

that our dojoConfig gives us?

Reply:
This was another auto import from my editor, will fix